spec/features/staff/fund_managers_can_invite_new_users_spec.rb

```diff
@@ -86,7 +90,9 @@
           click_button I18n.t("generic.button.submit")
 
           expect(page).to have_content(I18n.t("form.user.create.failed"))
-          expect(User.count).to eq(0)
+
+          expect(User.count).to eq(1) # This is the fund_manager
```

Comment:
Is there another way to express this expectation, this feels like it relies on prior knowledge of there only being one `User` that seems a bit tricky to follow? Can we expect the number of Users to not change in some way?

Perhaps we do not need the count check at all if we are confident that seeing the error means no User was created?

Reply:
Agree, I really hate this check but I couldn't think of a better way to do it!